avidemux_plugins/ADM_videoFilters6/asciiView/ADM_vidAscii.cpp

```diff
@@ -56,8 +56,8 @@ DECLARE_VIDEO_FILTER(   AsciiFilter,   // Class
                         ADM_UI_ALL,         // UI
                         VF_TRANSFORM,            // Category
                         "AsciiView",            // internal name (must be uniq!)
-                        "Ascii View",            // Display name
-                        "Ascii view" // Description
+                        QT_TRANSLATE_NOOP("Ascii View"),            // Display name
```

Comment:
You must supply a context, e.g.

```QT_TRANSLATE_NOOP("asciiView","Ascii View"), // Display name```

In this case "asciiView" is an arbitrary string serving as context for the translation (identical strings may be translated differently if they have different context).